### PR TITLE
fixing some things we missed

### PR DESCRIPTION
**Pull request by cmastr**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/dkp/konvoy/1.5/version-support/index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ navigationTitle: Version Support Policy
 title: Version Support Policy
 menuWeight: 280
 excerpt: Supported version policy for Konvoy
-beta: true
+beta: beta
```
Comment (dkoshkin) [inline pages/dkp/konvoy/1.5/version-support/index.md:7]: This doesn't look right

File: pages/dkp/kommander/1.3/install-airgapped/index.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ navigationTitle: Install air gapped
 title: Install Kommander air gapped
 menuWeight: 35
 excerpt: Install Kommander in an air gapped environment
-beta: false
+beta: true
```
Comment (dkoshkin) [inline pages/dkp/kommander/1.3/install-airgapped/index.md:7]: Is this really beta?

Comment (cmastr) [reply pages/dkp/kommander/1.3/install-airgapped/index.md:7]: according to @drewvanstone it is.

File: pages/dkp/konvoy/1.5/version-support/index.md (hunk 1 above)
Comment (cmastr) [reply pages/dkp/konvoy/1.5/version-support/index.md:7]: ha! typing too fast.

Comment (cmastr) [reply pages/dkp/konvoy/1.5/version-support/index.md]: fixed

File: pages/dkp/kommander/1.3/install-airgapped/index.md (hunk 2 above)
Comment (cmastr) [reply pages/dkp/kommander/1.3/install-airgapped/index.md:7]: Yeah, just checked with Sam B. It's still beta.

**Pull request merged**